app/tests/unit/layout/default-layout.test.tsx

```diff
@@ -1,10 +1,35 @@
 import React from 'react';
 import {shallow} from 'enzyme';
 import {DefaultLayoutComponent} from 'layouts/default-layout';
+import {defaultLayout_session} from '__generated__/defaultLayout_session.graphql';
 
 describe('The DefaultLayout component', () => {
-  it('matches the last accepted Snapshot', () => {
+  it('when logged out matches the last accepted Snapshot', () => {
     const wrapper = shallow(<DefaultLayoutComponent session={null} />);
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('when logged in matches the last accepted Snapshot', () => {
+    const session = {
```

Comment:
```suggestion
    const session: defaultLayout_session = {
```

This is a better option than casting. This way if you have an issue with the query, the correct line will be highlighted.

Reply:
yeah I get typescript errors just doing this as well, so this object needs to be changed, but I'm not quite sure how.